pyproject.toml

```diff
@@ -3,7 +3,8 @@ requires = ["setuptools",
             "setuptools_scm>=6.2",
             "wheel",
             "cython==0.29.22",
-            "jinja2==2.10.3",
+            "markupsafe==2.1.0",
```

Comment:
Can you add a comment here saying it's a jinja dep?

Reply:
Can I just do this or will this break TOML?

```diff
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -3,7 +3,7 @@ requires = ["setuptools",
             "setuptools_scm>=6.2",
             "wheel",
             "cython==0.29.22",
-            "markupsafe==2.1.0",
+            "markupsafe==2.1.0",  # jinja2 dependency
             "jinja2==3.0.3",
             "oldest-supported-numpy",
             "extension-helpers"]
```

Reply:
I can also just keep jinja 2.x and pin older markupsafe... if we cannot just remove this altogether as per #12558

Reply:
OK... I updated the pin and added comment, in case we still need this patch.